FAQ.md

```diff
@@ -0,0 +1,268 @@
+- **Versioning Before 1.0:** Until we reach version 1.0, breaking changes may be introduced as the specification evolves. Users should be prepared for potential changes during this period.
+
+For more details, see the [versioning and compatibility section](https://cdevents.dev/docs/primer/#versioning) of our documentation.
+
+---
+
+### **Q: Are there any limitations or known issues with CDEvents?**
+**A:** As with any specification, CDEvents has its limitations and ongoing areas of improvement. Current known issues and limitations are tracked in our [GitHub issues tracker](https://github.com/cdevents/spec/issues). We encourage users to report any issues they encounter and participate in discussions to help address them.
+
+---
+
+### **Q: How does CDEvents handle large-scale event processing?**
+**A:** CDEvents is designed with scalability in mind. Key features that support large-scale event processing include:
+- **Compact Event Design:** Events are kept as small as possible to minimize overhead and improve performance.
+- **Efficient Routing and Filtering:** The event context includes fields (such as `subject`) that can be used to efficiently route and filter events without parsing the entire payload. These fields are also available at the CloudEvents level, enabling even more efficient processing.
+- **Efficient Workflow Search:** CDEvents includes links to help efficiently search and retrieve events that belong to a specific workflow, supporting the storage and querying of large volumes of events.
+  
+While CDEvents provides the foundation for scalability, implementers should ensure their infrastructure (e.g., message brokers, databases) is capable of handling the expected event load based on their specific use case.
+
+---
+
+### **Q: Are there any case studies or success stories using CDEvents?**
+**A:** 
+1. **Fidelity Investments**: The CDEvents project is key to enabling interoperability, which simplifies the Software Delivery Lifecycle (SDL) processes. Fidelity uses an event-driven architecture to achieve continuous compliance for apps delivered in a highly regulated environment. Their internal tool, the Fidelity Pipeline Library, offers reusable pipeline templates with event triggers, showcasing how CDEvents can enhance developer experience and feature velocity.
+
+2. **Ericsson**: Ericsson co-founded CDEvents to address the need for innovative and simplified software deployment in telecommunications. They have implemented CDEvents in their CI/CD pipelines, demonstrating its effectiveness in achieving interoperability and streamlining processes.
+
+3. **TestKube**: This Kubernetes-native testing framework has adopted CDEvents to improve feedback loops and quality-gated workflows within CI/CD pipelines. The integration of test events allows for better tracking and management of testing activities.

```

Comment:
May I ask what is the source of this?

I don't think we should include statements about what companies do or do not do with CDEvents unless they are:
- either reviewed/provided by someone at the company
- or taken from some publicly available source

My recommendation would be for now to drop this question, and we can work on it on a separate PR. 
Alternatively please only include links to publicly available documents or statements, such as the Fidelity end-user story.

Reply:
few of the sources through which i had the reference :
- https://cd.foundation/blog/2023/05/08/cdevents-real-world-adoption/
- https://cd.foundation/tag/ericsson/
- https://testkube.io/blog/testkube-joins-cd-foundation

but as you said we'll drop this question for now and focus on it later on